### PR TITLE
Create Overview of work done AutSPACEs Project

### PR DESCRIPTION
**Pull request by sowmya9507**
I have updated and documented everything that I feel is important for the project. Please let me know if I need to update anything else :)

### CODE REVIEW

File: Overview/Overview of work done AutSPACEs Project.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+### Technical environment of the internship
+
+* Python - Django Web framework was used to make the interact pages with each other. Django is a Python-based free and open-source web framework that follows the model–template–views architectural pattern. It takes care of much of the hassle of web development, so you can focus on writing your app without needing to reinvent the wheel.
+
+* Web development - HTML/CSS/ Bootstrap was used to create the actual content of the page, such as written text, and CSS (Cascade Styling Sheets) is responsible for the design or style of the website, including the layout, visual effects and background color. Bootstrap is a framework to help you design websites faster and easier. It includes templates of HTML and CSS.
+
+* Git version control - In order to document everything, use the pre-exiting code, push the code or commit, the repository is being continuously used and Git commands are very useful.
+
+### List of hard/soft skills developed.
```
Comment (GeorgiaHCA) [inline Overview/Overview of work done AutSPACEs Project.md:116]: This is super useful @sowmya9507 thanks!

File: Overview/Basic Understanding of the MVP for AutSPACEs.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* The working issues can be found under the [Milsesones section](https://github.com/alan-turing-institute/AutSPACEs/milestones)
+* Go through the [MVP Requirements List](https://github.com/alan-turing-institute/AutSPACEs/blob/main/platform-design/MVP-requirements-list.md) before starting working on project
+* Follow the [Figma Wireframe](https://github.com/alan-turing-institute/AutSPACEs/blob/main/platform-design/MVP-requirements-list.md) for the design.
+
+## Requirements
+![](https://i.imgur.com/F48yYoV.png)
+
+![](https://i.imgur.com/SyXZ1y7.png)
+
+## Demo for Working of MVP is uploaded to this [folder](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/sowmya-handover-docs/Overview/AutSPACEs%20Screen%20Recording.mp4)
```
Comment (GeorgiaHCA) [inline Overview/Basic Understanding of the MVP for AutSPACEs.md:12]: This is fab thank you for doing this @sowmya9507 I'll take a closer look

File: Overview/Overview of work done AutSPACEs Project.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+* The next came splitting up the tasks into Static and dynamic components. I started creating the static pages according to the Figma wireframe. This page was designed with the help of HTML, Bootstrap and CSS. I was successfully able to understand and create a landing page, experiences page, Moderation page and also a sign up page (this was not part of the wireframe)
+
+**Landing Page**
+![](https://i.imgur.com/UIhIbJQ.png)
+
+**View Experiences Page**
+
+![](https://i.imgur.com/S1W18ti.png)
+
+** Moderation Page**
```
Comment (GeorgiaHCA) [inline Overview/Overview of work done AutSPACEs Project.md:45]: Tiny change: **Moderation Page** (extra gap)

**Review by GeorgiaHCA — APPROVED** [review]
Hi @sowmya9507 - this is a brilliant piece of work representing what you've done for AutSPACEs! It's especially useful to have the screenshots. The explanations are also really clear to me. I wonder if @lottycoupat has a view where we can put this so that it is most useful as a resource? 🎉 💛

**Comment by GeorgiaHCA** [comment]
Hey @sowmya9507 - I wonder if it would be worth uploading the video to YouTube and then using an embed link so that it is available on the site without being downloaded? What do you reckon? Just a thought :)

File: Overview/Overview of work done AutSPACEs Project.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+![](https://i.imgur.com/7ytGxki.png)
+
+# Brief report
+### Introduction
+
+AutSPACEs is a citizen science platform co-created by autistic people and their supporters alongside researchers and the open source community. The aim is to create an online space where autistic people can share their experiences around sensory processing and autism. We know that the autistic people have challenges navigating the world that was not build by Autistic people and for the autistic people. We would like to use citizen science n a participatory framework to rise up the autistic voices and educate non-autistic people on how they can change the behavior and also change the world. We would like them to understand how to be more inclusive and enabling the autistic people. Understanding sensory processing differences, and learning what could make environments better for autistic people, are top community priorities. In order to really make a difference, we want to take the research out of the lab and learn from the lived experience of autistic people in their daily lives.
```
Comment (lottycoupat) [inline Overview/Overview of work done AutSPACEs Project.md:64]: We would like to use citizen science **_in_** a participatory framework to rise up....

**Comment by lottycoupat** [comment]
I think as this is info on the project and the work that you have done, it could fit in well in the "Get-Involved" folder where you are giving an example of how someone has gotten involved and to the extent of it. What do you think? @GeorgiaHCA @sowmya9507

**Comment by GeorgiaHCA** [comment]
Yep, I think that works @lottycoupat ! Good idea.

**Comment by lottycoupat** [comment]
Hi @sowmya9507 love the work. Could you possibly put this in the following folders:

Overview of work done on AutSPACEs project - https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/applications/cri-internship 

Basic understanding of MVP for autSPACEs - https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/project-information

**Comment by BrainonSilicon** [comment]
Hey @GeorgiaHCA! Just wanted to check if this documentation was still in date and could be merged?

**Comment by GeorgiaHCA** [comment]
Hiya @BrainonSilicon yep thanks for checking! I think there is some reshuffling to do of where things go in the repo overall but def worth merging this first before it becomes a merge conflict!

**Pull request merged**